### ISSUE TITLE
URL (http://www.photoshow-gallery.com/) Compromised

### ISSUE THREAD

**Issue by Zelytic** [thibaud-rohmer/PhotoShow#385]
The URL http://www.photoshow-gallery.com appears to be compromised. It is forwarding me to https://9348fqe15x9ojblc39.miwgh.co.in/

**Comment by angelmankel** [comment]
^

**Comment by anandslab** [comment]
Yes and spreading malware. Too bad @thibaud-rohmer appears inactive. At the very least the link should be removed.

**Comment by DizzyOnion** [comment]
I found him on LinkedIn and dropped him a note. Let's see if that helps. 🤞

**Comment by thibaud-rohmer** [comment]
@DizzyOnion thank you so much for reaching out ! I have done the update.

**Comment by SimpleHomelab** [comment]
@thibaud-rohmer Thanks. Can you also update the "About" section of the repo? It still has the link. 

IMHO, the original message from @Zelytic must be updated to remove the compromised links. Essentially any reference to the compromised domain must be removed (unless you can get the domain back) so as to not provide any SEO juice or referrals to the domain.

**Comment by Zelytic** [comment]
@SimpleHomelab . I've removed the links from my message.

Thanks everyone!

**Comment by thibaud-rohmer** [comment]
@SimpleHomelab , good catch, "About" section updated :)
